packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/SubjectReductions/SubjectReductions.spec.js

```diff
@@ -0,0 +1,26 @@
+import { expect } from 'chai'
+import { reducedEmptySubject, reducedSubject } from './mocks'
+
+describe('Component > SubjectReductions', function () {
+  describe('with reductions', function () {
+    it('should exist', function () {
+      expect(reducedSubject).to.be.ok()
+    })
+
+    it('should have points', function () {
+      expect(reducedSubject.data.workflow.subject_reductions[0].data.points).to.be.a('array')
+      expect(reducedSubject.data.workflow.subject_reductions[0].data.points).not.to.be.empty
+    })
+  })
+
+  describe('without reductions', function () {
+    it('should exist', function () {
+      expect(reducedEmptySubject).to.be.ok()
+    })
+
+    it('should have points', function () {
+      expect(reducedEmptySubject.data.workflow.subject_reductions[0].data.points).to.be.a('array')
+      expect(reducedEmptySubject.data.workflow.subject_reductions[0].data.points).to.be.empty()
+    })
+  })
+})
```

Comment:
None of these tests run the `SubjectReductions` function and test its returned value.

Reply:
This was removed as part of the later PR incorporating the SubjectReductions hook.